SW_Flow_lib.c

```diff
@@ -124,25 +124,24 @@ static ST_RGR_VALUES stValues; // keeps track of the soil_temperature values
 /* --------------------------------------------------- */
 
 /**
-	\fn void veg_intercepted_water(double *pptleft, double *wintgrass, double ppt, double x, double scale, double a, double b, double c, double d) {
-	\brief Calculate the water intercepted by vegetation.
-
-	Equations based on Corbet and Crouse (1968). \cite Corbett1968
-
-  \param pptleft Amount of precipitation left after interception.
-	\param wintveg Amount of precipitation interception by veg.
-	\param ppt daily precipitation
-	\param x vegetation cover or LAI for the day (based on monthly biomass
-	       values, see the routine "initprod").
-	\param scale Scale paramater. Used to represent snow depth and fraction cover of vegtype.
-	\param a a parameter for intercept of grass interception equation.
-	\param b b parameter for intercept of grass interception equation.
-	\param c c parameter for slope of grass interception equation.
-	\param d d parameter for slope of grass interception equation.
-
-  \sideeffect Update values of global variables:
-  - pptleft Amount of precipitation left after interception.
-  - wintveg Amount of precipitation interception by veg.
+	@brief Calculate the water intercepted by vegetation.
+
+	Equations based on Corbett and Crouse 1968. @cite Corbett1968
+
+  @param *pptleft Remaining precipitation, (cm/day).

```

Comment:
Fix tabs in the sections so they are aligned in the correct spot.

Also no comma is need before units i.e. Remaining precipitation (cm/day)